client/upe-onboarding-wizard/index.js

```diff
@@ -7,12 +8,22 @@ import ReactDOM from 'react-dom';
 /**
  * Internal dependencies
  */
+import UpeToggleContextProvider from 'wcstripe/settings/upe-toggle/provider';
```

Comment:
Unfortunately the `UpeToggleContextProvider` lives in `wcstripe/settings/*`  and this file lives in `client/upe-onboarding-wizard/*`.

Since there are multiple PRs touching/needing `UpeToggleContextProvider`, I'm going to wait for the dust to settle a bit and then open a separate PR to move the file to a more shared location. This way I'm not disrupting other people's work.